src/components/commons/StatisticPanel.tsx

```diff
@@ -30,7 +33,11 @@ export function StatisticPanel({
       )}
     >
       <div className="card-bg rounded-[30px] py-10 px-6">
-        <div className="flex lg:flex-row flex-col gap-y-4">
+        <div
+          className={classNames("flex lg:flex-row flex-col gap-y-4", {
+            "md:flex-row": isTabletHorizontal,
```

Comment:
Have we checked if this is possible with pure CSS (grid/flex)? Using logical condition should be our last resort..